site/content/docs/getting-started/verify.en.md

```diff
@@ -11,8 +11,20 @@ The AWS Copilot CLI executables are cryptographically signed using PGP signature
 2. Retrieve the Amazon ECS PGP public key. You can use a command to do this or manually create the key and then import it.
     * Option 1: Retrieve the key with the following command.
     ```sh
-    gpg --keyserver hkp://keys.gnupg.net --recv BCE9D9A42D51784F
+    gpg --keyserver hkps://keyserver.ubuntu.com --recv BCE9D9A42D51784F
     ```
+    Historical note only: ~~`keys.gnupg.net`~~ does NOT exist any more.<br>
+
+    Notes to **ADVANCED** Users **ONLY**:
+    *   Let's all switch to use HKPS (HKP over TLS)
+    *   `keyserver.ubuntu.com` is only reachable via IPv4.
+    *   Other EXCELLENT Alternative keyservers are:
+        *   `hkps://keys.openpgp.org`
+        *   `hkps://pgp.mit.edu`
+        *   ~~`hkps://subkeys.pgp.net`~~<BR>
+    *   Since the release of GnuPG 2.1.9 (2015-10-09), the `--keyserver` option for `gpg` has been deprecated and users are recommended to use the `--keyserver` in  `~/.gnupg/dirmngr.conf` instead.<BR>See more in https://unix.stackexchange.com/a/683639
+    *   In the future, if you have to switch KeyServers (again), use this command to verify stored-key using this command 1st -> `gpg --keyserver ${KEYSERVER} --verbose --import-options show-only --recv BCE9D9A42D51784F`
+
```

Comment:
Alternatively, we can keep the notes but move them to an unexpanded drop-down:
```suggestion

    ???- note "Notes on key servers (optional)"

        * Some alternative key servers are:
            *   `hkps://keys.openpgp.org`
            *   `hkps://pgp.mit.edu`
        * To switch to a different key server, use this command to verify stored-key using this command first -> `gpg --keyserver ${KEYSERVER} --verbose --import-options show-only --recv BCE9D9A42D51784F`

```

I removed some points but kept ones that I felt like might be the most useful for users to complete the verification. Thank you very much for listing out the alternative servers! It's very helpful as sometimes some server go down and people have to switch to another.

Reply:
<img width="1067" alt="Screen Shot 2022-09-12 at 2 07 31 PM" src="https://user-images.githubusercontent.com/79273084/189758937-59a78901-e6cf-4799-8688-0bff3422e0cd.png">